manim/mobject/opengl/opengl_point_cloud_mobject.py

```diff
@@ -92,7 +92,7 @@ def thin_func(num_points=num_points):
 
     def set_color_by_gradient(self, *colors):
         self.rgbas = np.array(
-            list(map(color_to_rgba, color_gradient(*colors, self.get_num_points()))),
+            map(color_to_rgba, color_gradient_as_list(*colors, self.get_num_points())),
```

Comment:
Doing something like
```py
np.array(map(int, "34"))
```
returns `array(<map object at [address]>, dtype=object)` instead of `array([3, 4])`, which is what we want. We need to convert that `map` to a `list` before converting it to a NumPy array (another option would be using `np.fromiter`, but let's keep consistency with the rest of the code):

```suggestion
            list(map(color_to_rgba, color_gradient_as_list(*colors, self.get_num_points()))),
```